### PR TITLE
Homework Module 2 by ms00780

### PR DESCRIPTION
**Pull request by ms00780**

### CODE REVIEW

File: module2/homework/addEven/addEven.hpp [hunk 1]
```diff
@@ -2,7 +2,12 @@
...
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+     int sum = 0;
+      
+      for (int i = 0; i < numbers.size(); i ++) {
+          if (numbers[i] % 2==0){
+              sum=sum + numbers[i];
```
Comment (piezol) [inline module2/homework/addEven/addEven.hpp:9]:
```suggestion
          if (numbers[i] % 2 == 0) {
              sum = sum + numbers[i];
```

File: module2/homework/addEven/addEven.hpp [hunk 2]
```diff
@@ -2,7 +2,12 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+     int sum = 0;
+      
+      for (int i = 0; i < numbers.size(); i ++) {
```
Comment (piezol) [inline module2/homework/addEven/addEven.hpp:7]:
```suggestion
      for (int i = 0; i < numbers.size(); i++) {
```

File: module2/homework/generateSequence/generateSequence.hpp [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+  std::vector<int> vec;
+    vec.push_back (step);
+  for (int i = 1; i <= count; i ++){
+    vec.push_back(vec[i - 1] + step);
+  }
+ return vec;
```
Comment (piezol) [inline module2/homework/generateSequence/generateSequence.hpp:10]:
```suggestion
    std::vector<int> vec;
    vec.push_back (step);
    for (int i = 1; i <= count; i ++){
        vec.push_back(vec[i - 1] + step);
    }
    return vec;
```

File: module2/homework/maxOfVector/maxOfVector.hpp [hunk 4]
```diff
@@ -3,6 +3,13 @@
...
-    return {};
+    int max;
+       max = vec.front();
+       for (int i = 1; i < vec.size(); i++){
+           if (vec[i] > vec[i - 1]){
+               max = vec[i];
+       } else
+           i ++;
+     }
+    return max;
```
Comment (piezol) [inline module2/homework/maxOfVector/maxOfVector.hpp:14]: Pleaase use consistent indentation.
This function is basically unreadable.

File: module2/homework/nwdNww/nwdNww.hpp [hunk 5]
```diff
@@ -1,11 +1,22 @@
...
+       lhs = sec;
+     }
+    return lhs;
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+    return lhs/NWD(lhs, rhs)*rhs;
 }
```
Comment (nauka-programowania-MA) [inline module2/homework/nwdNww/nwdNww.hpp:22]: code formatting!

File: module2/homework/maxOfVector/maxOfVector.hpp (hunk 4 above)
Comment (ziobron) [reply module2/homework/maxOfVector/maxOfVector.hpp:14]: I agree. You can use code autoformatter. You can find how to configure automatic code formatting in VSC here: https://coders.school/edu/courses/powtorka-1-qa/lesson/formatowanie-kodu/

**Comment by ziobron** [comment]
+4 XP addEven
+6 XP NwwNwd
+5 XP maxOfVector
+5 XP generateSequence
+8 XP delivery before 31.05

{ "requirement_ids": [1119, 1631, 1119, 1633, 1119, 1634, 1119, 1635], "points": [2, 4, 2, 6, 2, 5, 2, 5] }

**Pull request closed without merging**